src/fontMetrics.js

```diff
@@ -183,14 +183,28 @@ export type CharacterMetrics = {
     width: number;
 };
 
+export type MetricMap = {
+    [string]: { [string]: number[] }
+}
+let metricMap = {};
+
+/**
+ * This function extends metricMap object with a new metric information
+ */
+export function addFontMetrics(newMetrics: MetricMap) {
+    metricMap = Object.assign(metricMap, newMetrics);
+}
+// add the default font metrics
+addFontMetrics(defaultMetricMap);
```

Comment:
I think we can just `import metricMap from "../submodules/katex-fonts/fontMetricsData";` above and not bother calling `addFontMetrics` here.

Reply:
Done